ci/infra/testrunner/testrunner.py

```diff
@@ -169,6 +170,7 @@ def main():
                            help="only list tests to be executed")
     test_args.add_argument("-v", "--verbose", dest="verbose", action="store_true", default=False,
                            help="show all output")
+    test_args.add_argument("--skip-setup", help="setup step to skip")
```

Comment:
shouldn't we add here the options for this argument so that it is documented and is not a free text?